### PR TITLE
Added OCurrent organisation to the list of adopters

### PR DESCRIPTION
**Pull request by rikusilvola**
We've adopted OCaml Code of Conduct for the whole OCurrent organisation. I'd like to add the organisation to the list of adopters.

### CODE REVIEW

File: list-of-adopters.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ following projects/spaces have adopted this Code of Conduct -
 * [ocsigen/ocsigen-toolkit](https://github.com/ocsigen/ocsigen-toolkit) in [ocsigen/ocsigen-toolkit#224](https://github.com/ocsigen/ocsigen-toolkit/pull/224)
 * [ocsigen/ocsigenserver](https://github.com/ocsigen/ocsigenserver) in [ocsigen/ocsigenserver#227](https://github.com/ocsigen/ocsigenserver/pull/227)
 * [ocsigen/tyxml](https://github.com/ocsigen/tyxml) in [ocsigen/tyxml#318](https://github.com/ocsigen/tyxml/pull/318)
+* [OCurrent organisation](https://github.com/ocurrent) see [the organisation's profile](https://github.com/ocurrent/.github)
```
Comment (Sudha247) [inline list-of-adopters.md:27]: Thanks! Do we still want to keep `current-bench` as a separate entry?

Comment (rikusilvola) [reply list-of-adopters.md:27]: Yes, since it has a different list of contacts.

Comment (Sudha247) [reply list-of-adopters.md:27]: Sounds good to me.

**Review by Sudha247 — APPROVED** [review]

File: list-of-adopters.md (hunk 1 above)
Comment (pitag-ha) [reply list-of-adopters.md:27]: Cool, it's also nice to know that GitHub gets the precedence of organization CoC vs repo CoC right: The "Code of conduct" link in the About section of the `current-bench` repo links to the repo CoC; the one in the other repos links to the organization CoC (it's something I've been wondering and now I know).

**Review by pitag-ha — APPROVED** [review]
Thank you! It's good to see that also more whole organizations start adopting the CoC.

**Pull request merged**